src/gps/GPS.cpp

```diff
@@ -767,7 +774,19 @@ GPS::~GPS()
 
 void GPS::setGPSPower(bool on, bool standbyOnly, uint32_t sleepTime)
 {
-    LOG_INFO("Setting GPS power=%d\n", on);
+    // Record the current state
```

Comment:
please don't use lego comments :-) The syslog facility does not process them well. Use sprintf-style replacements instead.

Reply:
Ah right, I wasn't aware, sorry. I've probably sprinkled a few more of those around the code-base that'll need tidying up, now that you mention it.